.idea/deployment.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we get a .gitignore entry for the .idea/ directory?

Reply:
This is probably just a vendoring artifact.

Reply:
Done